shotover-proxy/src/transforms/throttling.rs

```diff
@@ -22,6 +23,12 @@ pub struct RequestThrottlingConfig {
 
 impl RequestThrottlingConfig {
     pub async fn get_source(&self) -> Result<Transforms> {
+        if self.max_requests_per_second < nonzero!(50u32) {
```

Comment:
This should be written as a validate() implementation.
Failures in validation give better information to the user. e.g. Includes the chain name in the error and splits the error across multiple lines to make it more readable.